plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryConnectorSmokeTest.java

```diff
@@ -0,0 +1,93 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.bigquery;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.testing.BaseConnectorSmokeTest;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.TestingConnectorBehavior;
+import org.testng.annotations.BeforeClass;
+
+import java.util.regex.Pattern;
+
+import static io.trino.plugin.bigquery.BigQueryQueryRunner.BigQuerySqlExecutor;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestBigQueryConnectorSmokeTest
+        extends BaseConnectorSmokeTest
+{
+    protected BigQuerySqlExecutor bigQuerySqlExecutor;
+
+    @BeforeClass(alwaysRun = true)
+    public void initBigQueryExecutor()
+    {
+        this.bigQuerySqlExecutor = new BigQuerySqlExecutor();
+    }
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        return BigQueryQueryRunner.createQueryRunner(
+                ImmutableMap.of(),
+                ImmutableMap.of("bigquery.metadata-cache-ttl", "5m"));
+    }
+
+    @SuppressWarnings("DuplicateBranchesInSwitch")
+    @Override
+    protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
+    {
+        switch (connectorBehavior) {
+            case SUPPORTS_TOPN_PUSHDOWN:
+                return false;
+
+            case SUPPORTS_RENAME_SCHEMA:
+                return false;
+
+            case SUPPORTS_RENAME_TABLE:
+                return false;
+
+            case SUPPORTS_ADD_COLUMN:
+            case SUPPORTS_RENAME_COLUMN:
+                return false;
+
+            case SUPPORTS_NOT_NULL_CONSTRAINT:
+                return false;
+
+            case SUPPORTS_TRUNCATE:
+                return true;
+
+            case SUPPORTS_NEGATIVE_DATE:
+                return false;
+
+            default:
+                return super.hasBehavior(connectorBehavior);
+        }
+    }
+
+    @Override
+    public void testShowCreateTable()
+    {
+        assertThat((String) computeScalar("SHOW CREATE TABLE region"))
+                .matches(format(
+                        "CREATE TABLE %s.%s.region \\(\n" +
+                                "   regionkey bigint NOT NULL,\n" +
+                                "   name varchar NOT NULL,\n" +
+                                "   comment varchar NOT NULL\n" +
+                                "\\)",
+                        Pattern.quote(getSession().getCatalog().orElseThrow()),
+                        Pattern.quote(getSession().getSchema().orElseThrow())));
+    }
+}
```

Comment:
Do we want an additional test method here that:

- Create schema using Trino (so that dataset gets cached)
- List schemas using Trino and verify created schema is visible
- Create a table in the schema/list tables in the schema using Trino
- Drop dataset using BigQuery
- List schemas using Trino - notice that schema is still visible
- Try to access table/list tables in schema using Trino - get an error

This would verify that the cache is indeed being used  and would prevent reoccurence of https://github.com/trinodb/trino/pull/10725